portal-kernel/src/com/liferay/document/library/kernel/service/persistence/packageinfo

```diff
@@ -1 +1 @@
-version 6.0.0
\ No newline at end of file
+version 7.0.0
```

Comment:
I think that :
`portal-kernel/src/com/liferay/document/library/kernel/service/packageinfo`
is needed to upgrade too

Can you check it?